public/css/main.stylesheet.css

```diff
@@ -164,6 +164,10 @@ input[type="text"]:focus {
   color: white;
 }
 
+label {
+  cursor: pointer;
```

Comment:
This will effect the all labels in the application? Is this intentional? 
prefer to use a class for these.

Reply:
it kind of is intentional

Reply:
@nitin10s

Reply:
ok I see, done it.